docs/zh_CN/HowToImplementTrainingService.md

```diff
@@ -0,0 +1,154 @@
+# **How to Implement TrainingService in NNI**
```

Comment:
why this English version doc in zh_CN?

Reply:
> why this English version doc in zh_CN?

It's a placeholder, and not translated yet.